tri.js

```diff
@@ -2,60 +2,441 @@
 
 // variables
 let gameStarted = false;
-var circleArrayGreen = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
-var circleArrayBlack = [];
 var resetButton = document.querySelector("#reset");
-var circleOne = document.querySelector("#one");
+var circleOne = document.querySelector("#\\31 ");
 var allCircles = document.querySelectorAll(".circle");
+var moveCount = 0;
+var x = "";
+var y = "";
+var r = "";
+var movesByPlay = [];
+var startClick = false;
+
+//decision tree - first 3 moves
+var moves = [
```

Comment:
I maintain that this will eventually become prohibitively large. At lease move this to a separate file to keep this file from getting too large. Look in to [require](https://requirejs.org/) or [modules](https://jsmodules.io/) to do this.